bson/doc.go

```diff
@@ -22,11 +23,13 @@
 //
 // Native Go Types
 //
-// This D and M types defined in this package can be used to build concise representations of BSON using native Go
-// types.
+// This D and M types defined in this package can be used to build concise of BSON using native Go types. D is a slice
```

Comment:
No longer grammatical.  Concise has to modify something.

Reply:
Whoops. Changed to "The D and M types defined in this package can be used to build representations of BSON using native Go types"